atprototools/__init__.py

```diff
@@ -115,8 +115,19 @@ def get_skoot_by_url(self,url):
         )
 
         return resp
+    
+    def uploadBlob(self, blob_path, content_type):
+        headers = {"Authorization": "Bearer " + self.ATP_AUTH_TOKEN, "Content-Type": content_type}
+        with open(blob_path, 'rb') as f:
+            files = {'file': f}
```

Comment:
Here's the problem I think -- was uploading a file pointer / python file object, probably the string.

Reply:
Changed to this to fix: 
```python
        with open(blob_path, 'rb') as f:
            image_bytes = f.read()
            resp = requests.post(
                self.ATP_HOST + "/xrpc/com.atproto.repo.uploadBlob",
                data=image_bytes,
                headers=headers
            )
```

Reply:
Python bytes object containing the byte content of the image file.

```
>>> f = open("tests/test.jpeg", 'rb')
>>> c = f.read()
>>> type(c)
<class 'bytes'>
>>> c[0:20]
b'\xff\xd8\xff\xe1\x00\xbcExif\x00\x00II*\x00\x08\x00\x00\x00'
```